src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unnecessaryparentheses/InputUnnecessaryParenthesesConditionalExpression.java

```diff
@@ -0,0 +1,31 @@
+/*
+UnnecessaryParentheses
+tokens = (default)EXPR, IDENT, NUM_DOUBLE, NUM_FLOAT, NUM_INT, NUM_LONG, \
+         STRING_LITERAL, LITERAL_NULL, LITERAL_FALSE, LITERAL_TRUE, ASSIGN, \
+         BAND_ASSIGN, BOR_ASSIGN, BSR_ASSIGN, BXOR_ASSIGN, DIV_ASSIGN, \
+         MINUS_ASSIGN, MOD_ASSIGN, PLUS_ASSIGN, SL_ASSIGN, SR_ASSIGN, STAR_ASSIGN, \
+         LAMBDA, TEXT_BLOCK_LITERAL_BEGIN, LAND, LITERAL_INSTANCEOF, GT, LT, GE, \
+         LE, EQUAL, NOT_EQUAL, UNARY_MINUS, UNARY_PLUS, INC, DEC, LNOT, BNOT, \
+         POST_INC, POST_DEC, QUESTION
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.unnecessaryparentheses;
+
+public class InputUnnecessaryParenthesesConditionalExpression {
+    void method() {
+        int w = 5;
+        int x = (w == 3) ? (3) : (4);
+        // 3 violations above
+        //  'Unnecessary parentheses around expression'
+        //  'Unnecessary parentheses around expression'
+        //  'Unnecessary parentheses around expression'
```

Comment:
The last 2 messages should be: 
```
Unnecessary parentheses around literal '3'.
Unnecessary parentheses around literal '4'.
```

I think the messages content is not being checked by our parser because you missed `:` after `// 3 violations above`. It should be: 

```
        // 3 violations above:
        //                'Unnecessary parentheses around expression'
        //                'Unnecessary parentheses around literal '3'.
        //                'Unnecessary parentheses around literal '4'.
```

please fix all where possible

Reply:
done